pkg/broker/broker.go

```diff
@@ -355,13 +377,8 @@ func (a AnsibleBroker) Bind(instanceUUID uuid.UUID, bindingUUID uuid.UUID, req *
 	//
 	// See if the service instance still exists, if not send back a badrequest.
 
-	instance, err := a.dao.GetServiceInstance(instanceUUID.String())
+	instance, err := a.GetServiceInstance(instanceUUID)
```

Comment:
I can name it something else/put it somewhere else, there was just a lot of duplicate logic for getting the service instance and checking for etcd not found errors, just wanted to pull it out to a function rather than copy paste it yet again.